orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/job/RunJobStageContext.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.orca.clouddriver.pipeline.job;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.netflix.spinnaker.orca.clouddriver.pipeline.job.model.JobStatus;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RunJobStageContext {
+  private JobStatus jobStatus;
+  private String cloudProvider;
+  private String cloudProviderType;
+  private String application;
+  private String credentials;
+
+  @JsonIgnore private Map<String, Object> other = new HashMap<>();
+
+  @JsonAnyGetter
+  public Map<String, Object> other() {
+    return other;
+  }
+
+  @JsonAnySetter
+  public void set(String name, Object value) {
+    other.put(name, value);
+  }
+
+  public JobStatus getJobStatus() {
```

Comment:
Any reason not to annotate the class with `@Getter` and `@Setter` instead of manually defining these?

Reply:
Do you know if this works `@JsonAnyGetter` for the `other()` properties?  That's why I didn't do it here

Reply:
I don't know for sure; my guess is that it would work because serialization/deserialization shouldn't care how the `getXXX` and `setXXX` methods were generated.  I think it just looks for methods with those names, then if absent would fall back to the `@JsonAny...` annotated methods.  But I also am not 100% sure of this so definitely could be wrong here...

Reply:
I'm going to leave this as it is for now, although I think you may be right.